src/dms/core/EdFi.DataManagementService.Core/EdFi.DataManagementService.Core.csproj

```diff
@@ -9,8 +9,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Content Include="EdFi.DataManagementService.Core.nuspec" CopyToPublishDirectory="Always" CopyToOutputDirectory="Never" />
-    <PackageReference Include="EdFi.DataStandard51.ApiSchema" />
     <PackageReference Include="Be.Vlaanderen.Basisregisters.Generators.Guid.Deterministic" />
+    <PackageReference Include="EdFi.DataStandard52.ApiSchema.Core" />
```

Comment:
What does the `.Core` mean?

Reply:
I thought EdFi.DataStandard52.ApiSchema.Core is core package for apischema , so I kept the name .